packages/api/lib/rulesHelpers.js

```diff
@@ -558,13 +557,13 @@ async function invokeRerun(rule) {
   await invoke(process.env.invoke, payload);
 }
 
-/*
+/**
  * Updates rule trigger for rule
  *
- * @param {ApiRule}        original- Rule to update trigger for
- * @param {ApiRule}        updates - Updates for rule trigger
- * @param {Knex}           knex    - Knex DB Client
- * @returns {ApiRule}              - Returns new rule object
+ * @param {RuleRecord} original - Rule to update trigger for
+ * @param {Object} updates      - Updates for rule trigger
```

Comment:
Oh interesting. So defining fields on an ad-hoc object if we're not going to create a type. This would be good to include in the instructions for devs when we make typing a more formal requirement on tickets.

https://github.com/nasa/cumulus/pull/3149/commits/d3e0a5ffc896d1e7f34a18e3fb75c033c9348e87

Reply:
That's not a TS thing, that's a JSDoc thing: https://jsdoc.app/tags-param.html#parameters-with-properties